examples/kusama/assetHub/bridgeTransfers/assetHubDOTToPolkadot.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * When importing from @substrate/asset-transfer-api it would look like the following
+ *
+ * import { AssetTransferApi, constructApiPromise } from '@substrate/asset-transfer-api'
+ */
+import { AssetTransferApi, constructApiPromise } from '../../../../src';
+import { TxResult } from '../../../../src/types';
+import { GREEN, PURPLE, RESET } from '../../../colors';
+
+/**
+ * In this example we are creating a `polkadotXcm` pallet `transferAssets` call to send 1 DOT (foreign asset with location `{"parents":"2","interior":{"X1":{"GlobalConsensus":"Polkadot"}}}`)
+ * from a Kusama Asset Hub (System Parachain) account
+ * to a Polkadot Asset Hub account, where the `xcmVersion` is set to 4 and no `weightLimit` option is provided declaring that
+ * the tx will allow unlimited weight to be used for fees.
+ *
+ * NOTE: To specify the amount of weight for the tx to use provide a `weightLimit` option containing desired values for `refTime` and `proofSize`.
+ */
+const main = async () => {
+	const { api, specName, safeXcmVersion } = await constructApiPromise('wss://kusama-asset-hub-rpc.polkadot.io');
+	const assetApi = new AssetTransferApi(api, specName, safeXcmVersion);
+
+	let callInfo: TxResult<'call'>;
+	try {
+		callInfo = await assetApi.createTransferTransaction(
+			`{"parents":"2","interior":{"X1":{"GlobalConsensus":"Polkadot"}}}`,
```

Comment:
The API is weird to assume Polkadot **Asset Hub** when the `destChainId: string` provided here is an XCM Location defining Polkadot **Relay Chain**.

PAH should be correctly expressed as:
```diff
-			`{"parents":"2","interior":{"X1":{"GlobalConsensus":"Polkadot"}}}`,
+ 			`{"parents":"2","interior":{"X2":{"GlobalConsensus":"Polkadot", "Parachain": 1000}}}`,
```

Reply:
good catch, the API shouldn't make this assumption.

Reply:
Great point. I'll update the API to be based on the location specifically passed and remove any assumptions around AssetHub.